src/tests/functional/plugin/shared/src/snippets/mha.cpp

```diff
@@ -55,7 +55,6 @@ void MHABase::SetUp() {
 
     inType = outType = prc;
     setInferenceType(prc);
-    init_thresholds();
```

Comment:
Could you clarify please why you removed this method in `MHA`-related tests?

Reply:
@v-Golubev asked to remove in both places

Reply:
Reverted

Reply:
@a-sidorova my point is that we already call this virtual method in SubgraphBaseTest::compare:
https://github.com/openvinotoolkit/openvino/blob/9817b839b5f15bba4ebae72999cf64c2f1b0d440/src/tests/functional/shared_test_classes/src/base/ov_subgraph.cpp#L250-L257

So we have the overridden methods and call them 😄 There are no point in calling the `init_threshold()` one more time in `SetUp`.

Reply:
@v-Golubev now I see. Thank you for the detailed explanation! Before the reason was unclear. Now I agree with removing a call of this method in `SetUp`, of course